io.gitlab.news_flash.NewsFlash.json

```diff
@@ -23,14 +23,48 @@
         }
     },
     "modules" : [
+        {
+            "name": "blueprint-compiler",
+            "buildsystem": "meson",
+            "sources": [
+                {
+                    "type": "git",
```

Comment:
there should be a cleanup rule for "*" to remove all blueprint after the build.

Reply:
I'm not following. Isn't the whole build directory be deleted after a build anyway?

Reply:
it's installed to run. 

it's even in the doc
https://jwestman.pages.gitlab.gnome.org/blueprint-compiler/flatpak.html

Reply:
cleanup remove installed files that are not necessary like build time only. this can include headers, static libraries, and also unreachable documentation.

Reply:
done, thx